nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -0,0 +1,351 @@
+          date-field="dc:created"
+          where='{"ecm:isRecord": "true"}'
+          metrics="cardinality(ecm:uuid)"
+          data="{{activeRetentionCount}}"
+          index="[[index]]"
+        >
+        </nuxeo-repository-data>
+
+        <!-- Number of Files under legal hold -->
+        <nuxeo-repository-data
+          start-date="[[_formatDate(startDate)]]"
+          end-date="[[_extendEndDate(endDate)]]"
+          date-field="dc:created"
+          ecm-mixin-type="HasRelatedText"
+          where='{"ecm:hasLegalHold": "true"}'
+          metrics="cardinality(ecm:uuid)"
+          data="{{legalHoldCount}}"
+          index="[[index]]"
+        >
+        </nuxeo-repository-data>
+
+        <nuxeo-card class="infoCard">
+          <div class="rangeSelector">
+            <span class="mainRetentionInfo"
+              >[[i18n('retentionAnalytics.underActiveRetention')]]
+              <span class="retentionCount">[[activeRetentionCount]]</span>
+            </span>
+            <span class="mainRetentionInfo"> | </span>
+            <span class="mainRetentionInfo"
+              >[[i18n('retentionAnalytics.legalHoldCount')]]
+              <span class="retentionCount">[[legalHoldCount]]</span>
+            </span>
+          </div>
+          <div class="rangeSelector">
+            <div class="dateLabel">
+              <label>[[i18n('retentionAnalytics.from')]]</label>
+            </div>
+            <nuxeo-date-picker value="{{startDate}}"></nuxeo-date-picker>
+            <div class="dateLabel">
+              <label>[[i18n('retentionAnalytics.to')]]</label>
+            </div>
+            <nuxeo-date-picker value="{{endDate}}"></nuxeo-date-picker>
+          </div>
+        </nuxeo-card>
+
+        <div class="flex-layout">
+          <!-- Retention by Document Type -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:primaryType"
+            where='{"ecm:isRecord": "true"}'
+            group-limit="20"
+            data="{{allDocCategoriesCount}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card heading="[[i18n('retentionAnalytics.retentionByDocType')]]">
+            <chart-pie
+              values="[[_values(allDocCategoriesCount)]]"
+              labels="[[_labels(allDocCategoriesCount)]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
+            >
+            </chart-pie>
+          </nuxeo-card>
+
+          <!-- RETENTION STATUS -->
+          <!-- No Retention -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            ecm-mixin-type="HasRelatedText"
```

Comment:
Do we need to specify and filter on the `mixin-type`

Reply:
@RSalem07 sorry I am not sure I understood your question

Reply:
Yes, the mixin-type `HasRelatedText` is a filed where we store some extra info like description, comment... for a given document. The purpose is to store text that will be queried by Nuxeo Elastic Search... 

Here we filter on this mixin type, I don't know if it's a requirement or why it's was done.

Reply:
@RSalem07 to be honest, when I remove ecm-mixin-type="HasRelatedText" I get many more documents, but some of them seem to be weird things like system files. I need to investigate more.

Reply:
Do you have more information on this point. Don't hesitate to reach Julien / Greg, to get more infos. I don't recall if the type filter was on the demo or added by you.
thx

Reply:
@RSalem07 researching about ecm-mixin-type it looks like it is not necessary AFAIK so I removed it.